db/migrate/20200924072906_establish_tenant_zero.rb

```diff
@@ -0,0 +1,33 @@
+class EstablishTenantZero < ActiveRecord::Migration[6.0]
+  def up
+    execute "INSERT INTO libraries (#{LIBRARY.keys.join(', ')}, created_at, updated_at)"\
+            " VALUES (#{LIBRARY.values.map(&connection.method(:quote)).join(', ')}, NOW(), NOW())"
+
+    TABLES.each do |table|
+      execute "UPDATE #{table} SET library_id = #{library_id} WHERE library_id IS NULL"
+    end
+  end
+
+  def down
+    TABLES.each do |table|
+      execute "UPDATE #{table} SET library_id = NULL WHERE library_id = #{library_id}"
+    end
+
+    execute "DELETE FROM libraries WHERE id = #{library_id}"
+  end
+
+  private
+
+  def library_id
+    @library_id ||= select_value("SELECT id FROM libraries WHERE hostname = #{connection.quote(LIBRARY[:hostname])}").presence ||
+      (raise "Error updating #{LIBRARY[:name]} record")
+  end
+
+  LIBRARY = {
+    name: "Chicago Tool Library",
+    hostname: "chicagotoollibrary.herokuapp.com",
+  }.freeze
+  TABLES = %w[users members memberships items loans].freeze
```

Comment:
We should add the extra tables from `db/migrate/20200925072325_tenantize_some_more.rb` here and rename this migration to come after that one.